package.json

```diff
@@ -118,7 +118,9 @@
     "memcached": "2.2.2",
     "mkdirp": "^1.0.4",
     "mocha": "^8.4.0",
+    "mocha-junit-reporter": "^2.0.2",
```

Comment:
We need junit because it logs file names. Junit is also recommended by CircleCI for Node.js (and probably general JS) projects.

Reply:
Or phrased the other way around: If we want to use mocha-junit-reporter over the xunit reporter, then we should remove the xunit reporter. Having both mocha-junit-reporter and xunit seems redundant.

You mention file names as the reason for this new reporter. I agree that it would be convenient to have them, but for what exactly are they required? My guess is they are required for the split-by=timings feature, how exactly does that work. I don't see the `results-junit.xml` files being fed back into the `circleci tests split --split-by=timings` command, does that command use those files by default?

Reply:
This discussion is now obsolete since we agreed on not going for the split-by=timings variant. Too little value for too much hassle (plus additional maintenance overhead down the road)